### DISCUSSION TITLE
Churning Away

### DISCUSSION THREAD

**Discussion by Acewiza** [Security-Onion-Solutions/securityonion#10242] [2.4] [answered]
New guy's running neck-and-neck with older brother in terms of alerting, writing almost 40 gig in 48 hours to the raid volume which doesn't show up in Influxdb, for some reason.

**Comment by Acewiza** [comment]
Nothing really unusual noted in the past couple weeks.  I did notice the fetch limit on alerts page seems confused:

![twopointfourfetchlimit](https://github.com/Security-Onion-Solutions/securityonion/assets/76695490/9f6f73a8-9eda-4dfe-b224-ad10d2641587)

**Reply by dougburks** [reply]
> I did notice the fetch limit on alerts page seems confused

What exactly do you mean?

**Reply by Acewiza** [reply]
It asks for 200 or whatever, but only ever returns 25.

**Reply by Acewiza** [reply]
Tell Wes I said hi!  Ask if he remembers me from that [Interloken](https://www.google.com/search?client=firefox-b-1-e&q=denver+interlokken) conference back in the day...

**Reply by dougburks** [reply]
> It asks for 200 or whatever, but only ever returns 25.

Looking at your screenshot, it's showing the first 25. You should be able to click the right arrow to go to the next 25. You should also be able to click the 25 and select a higher number in the drop-down box like 50.

Also note that it says `1-25 of 67` so it seems that there are only 67 rows of alerts returned within whatever time frame you are searching. If you search a longer time frame, you should get more than 67 rows of alerts.

**Reply by Acewiza** [reply]
It would appear I am the one confused by the terms "fetch" and "display."  ;-)  I must've set the display limit years ago and forgot about it.

**Comment by Acewiza** [comment]
Seems pretty stable - up for 26 days.  Might be the 1st Onion of mine that never got re-booted.  Not even once.

**Comment by Acewiza** [comment, marked as answer]
Beta 3!
